.github/workflows/unit.yml

```
@@ -0,0 +1,15 @@
+---
+name: Unit test validation
```

```suggestion
name: Unit test validation
```

This feels like quite a nit, but it would make the workflow name consistent with all the others ones in this repo (and I think other SubM repos).